examples/step-49/doc/intro.dox

```diff
@@ -99,13 +99,337 @@ many cases it is best to combine both approaches. The file can be easily
 reloaded by pressing "reload" under the "Geometry" tab if you want to write
 it by hand and see the effects in the graphical user interface of gmsh.
 
+//------------------------------------------------------------------------------
+// Informal edits start here
+
+<h4>Gmsh tutorial</h4>
+
 This tutorial contains an example <code>.geo</code> file that describes a box
-with two objects cut out in the interior. This is how
+with two circles cut out in the interior and several slits on the sides. This is how
 <code>example.geo</code> looks like in %Gmsh (displaying the boundary
 indicators as well as the mesh discussed further down below):
 
+// change picture once we show boundary IDs on paraview
 <img src="https://www.dealii.org/images/steps/developer/step-49.gmsh_picture.png" alt="">
 
+We will go through the basics of gmsh and show how we can obtain a mesh like this.
+Below is the documented <code>example.geo</code> file that creates the mesh.
+
+OpenCASCADE is CAD kernel that provides the geometry of model entities
```

Comment:
%Gmsh supports different kernels for geometry handling. We keep this as the default: